agent/src/beerocks/slave/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
 
 file(GLOB beerocks_agent_sources 
     ${MODULE_PATH}/backhaul_manager/*.c*
+    ${MODULE_PATH}/link_metrics/*.c*
```

Comment:
This is classic BPL implementation, so I would prefer it in the BPL.
But since you say below that this is a temporary solution, and this code should be refactored pretty soon so its part of the discovery service - (nitpick)

However, please add a comment here that this should move to the discovery agent and add a link to the github relevant task.

Reply:
Done.
I've created "[TASK] Move link metric related classes to BPL #910"
However, link metric collection is not meant to be part of the discovery service but it requires the topology database that will be populated by the discovery service.